src/govuk/components/hide-this-page/_index.scss

```
@@ -28,4 +28,14 @@
       display: none;
     }
   }
+
+  .govuk-hide-this-page-overlay {
```

It's not critical but I'd prefer we follow BEM here for consistency:
```suggestion
  .govuk-hide-this-page__overlay {
```

---

This is a thing that's confused me a little because, by my understanding, the way being suggested isn't following BEM. `overlay` is *not* a descendent element of `govuk-hide-this-page`, so it shouldn't use `__` as a separator. 

This is how it is elsewhere in Frontend though (we have `govuk-input` nested inside of `govuk-input__wrapper`) so I'm happy to change it for consistency.

---

You're right that it's not a descendant so we're not following BEM properly. My reading though is that it's part of the `govuk-hide-this-page` "suite" of component styles. Additionally as you say it's a consistent pattern within the rest of frontend. I wonder if this is something worth discussing in a dev catchup in terms of if we're using BEM correctly?

PS: Somewhen we need to sweep the branch and change hide this page to exit this page. I've been putting it off until these spikes are put to bed.